### DISCUSSION TITLE
Create a shotcut on my launcher for a catima-card?

### DISCUSSION THREAD

**Discussion by sorryusernameisalreadytaken** [CatimaLoyalty/Android#722] [Q&A] [answered]
Hey there. I use some cards very often and I want to add some of them directly to my launcher homescreen. Is that possible?

**Comment by Altonss** [comment, marked as answer]
> Hey there. I use some cards very often and I want to add some of them directly to my launcher homescreen. Is that possible?

Yes this is possible! You can add a card widget for a specific card on your homescreen.
The same feature but for opening groups is also in the works https://github.com/TheLastProject/Catima/pull/669
